qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/JavadocEmptyLineCheck/Invalid.java

```diff
@@ -0,0 +1,134 @@
+/**
```

Comment:
@dskalenko we also need tests for Javadoc above package:http://stackoverflow.com/questions/624422/how-do-i-document-packages-in-java